### ISSUE TITLE
Bold and italics highlighting when there is a space before closing *

### ISSUE THREAD

**Issue by GenaBitu** [Ionaru/easy-markdown-editor#74]
**Describe the bug**
When there is a space before the closing * of a part of the text, it shouldn't be italics/bold. In the editor, it not only is higlighted as bold, but the bolding doesn't end with the closing * and continues on.

**To Reproduce**
1. Type `**Hello ** World`

**Expected behavior**
\*\*Hello \*\* World

**Actual behaviour**
**\*\*Hello \*\* World**

** Version information **
- OS: Debian testing
- Browser: Firefox, Chrome
- EasyMDE version: 2.5.1

**Comment by GenaBitu** [comment]
codemirror/CodeMirror#5828

**Comment by roydukkey** [comment]
Spec: https://spec.commonmark.org/0.26/#example-328

**Comment by GenaBitu** [comment]
Yeah, as far as I understand it the bug is that it does not follow the spec. But seeing from the linked issue in CodeMirror, it looks like it won't be fixed...

**Comment by Ionaru** [comment]
Maybe they'll have it fixed CodeMirror 6.

**Comment by RudeySH** [comment]
Any chance we could get a branch of EasyMDE that uses CodeMirror 6?
